### PR TITLE
Add the information about the training dataset license

### PR DESCRIPTION
**Pull request by feihugis**
This PR collects information about the license of the training dataset.

### CODE REVIEW

**Comment by MLnick** [comment]
I'm just wondering why we put this info in the `assets/README` rather than the main `README`?

**Comment by feihugis** [comment]
@MLnick In the beginning, the information is put here for archiving the data license info we found from Kaggle. I think it can also be moved to the main `README`. Which place do you think is better?

**Comment by MLnick** [comment]
@feihugis let's rather put it in the main README, just under the LICENSE table

**Comment by feihugis** [comment]
@MLnick This part has been moved to the main README via https://github.com/IBM/MAX-Nucleus-Segmenter/pull/8/commits/3046e2f8ae5b8280f4e95425e9d4a7a448fa5984. 

@bdwyer2 The package number versions are also added in this PR (https://github.com/IBM/MAX-Nucleus-Segmenter/pull/8/commits/7c139b1e9533c90d21e3869ddb35b67727f0ca4e).

File: requirements.txt [hunk 1]
```diff
@@ -1,12 +1,12 @@
-numpy
-scipy
-Pillow
-cython
-matplotlib
-scikit-image
+numpy==1.16.2
+scipy==1.2.1
+Pillow==5.1.0
```
Comment (bdwyer2) [inline requirements.txt]: Can we use `6.0.0`?

Comment (bdwyer2) [reply requirements.txt]:
```suggestion
Pillow==6.0.0
```

File: requirements.txt [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+cython==0.28.2
```
Comment (bdwyer2) [inline requirements.txt]:
```suggestion
cython==0.29.6
```

File: requirements.txt [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
+matplotlib==2.2.2
+scikit-image==0.15.0
 tensorflow>=1.3.0
 keras>=2.0.8
```
Comment (bdwyer2) [inline requirements.txt]: I think we should also lock tf and keras to specific versions

Comment (feihugis) [reply requirements.txt]: @bdwyer2 Yes. The versions of tf and keras are specified now via https://github.com/IBM/MAX-Nucleus-Segmenter/pull/8/commits/0759b12b3c4685d8c3cc3ecc51c4e465532d2924.

**Comment by feihugis** [comment]
> LGTM pending travis CI

The tests pass. Will merge this PR.

**Pull request merged**